.github/workflows/release.yml

```diff
@@ -58,3 +62,30 @@ jobs:
               -H "Authorization: token ${TRAVIS_API_TOKEN}" \
               -d "$BODY" \
               "https://api.travis-ci.com/repo/${GH_ORGANIZATION_NAME}%2F${GH_REPOSITORY_NAME}/requests"
+
+  release-docker:
+    env:
+      REPO_NAME: datree
+      IMAGE_NAME: datree
+    name: Release container image to public registries
+    needs: release
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: docker/setup-buildx-action@v1
+    - name: Log in to DockerHub
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_TOKEN }}
+    - name: Build and push to registries
+      uses: docker/build-push-action@v2
+      with:
+        context: .
+        platforms: linux/amd64
+        push: true
+        tags: |
+          ${{ env.REPO_NAME }}/${{ env.IMAGE_NAME }}:latest
+          ${{ env.REPO_NAME }}/${{ env.IMAGE_NAME }}:${{ steps.define_version.outputs.version }}
```

Comment:
I think this call to a Step output fill fail since that is scoped to the `release` Job. I believe we would need to create an output var for the `release` Job itself and set it equal to our Step's output like:
```
release:
  outputs:
    version: ${{ steps.define_version.outputs.version }}
```

Then within the `release-docker` Job we'd use 
```${{ env.REPO_NAME }}/${{ env.IMAGE_NAME }}:${{ needs.release.outputs.version }}```

reference: https://stackoverflow.com/questions/59175332/using-output-from-a-previous-job-in-a-new-one-in-a-github-action/61236803#61236803